src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import { type FC, useState } from "react";
+import { Button, Stack, Typography } from "@mui/material";
+import { signOut } from "next-auth/react";
+import { useRouter } from "next/router";
+
+import { text } from "~/assets/text/index.js";
+import { styles } from "./styles.js";
+import {
+  BotBustersIcon,
+  SoundOffIcon,
+  SoundOnIcon,
+  UserIcon,
+} from "~/assets/icons";
+import { MenuButton } from "~/components/main-menu/menu-button.jsx";
+import { MainMenu } from "../main-menu/index.js";
+import { pages } from "~/router.js";
+
+interface Props {
+  isVerifiedUser: boolean;
+  isGamePlayed: boolean;
+  username: string;
+  open: boolean;
+  setOpen: (open: boolean) => void;
+  disconnect: () => Promise<void>;
+  points: number;
+}
+
+export const Navbar: FC<Props> = ({
+  isVerifiedUser,
+  isGamePlayed,
+  username,
+  open,
+  setOpen,
+  disconnect,
+  points,
+}) => {
+  const router = useRouter();
+  const [soundOn, setSoundOn] = useState(true);
+  const isVerifiedUserAndPlayed = isVerifiedUser && isGamePlayed;
+  const title = isVerifiedUserAndPlayed ? username : text.general.dailyScore;
+
+  const logout = async () => {
```

Comment:
Since it's referring to the action of logging out and not just a static logout component this should be called "logOut" (action). (I know this is very nitty picking hehe)

Reply:
can do, though personally when I read logout I always read action